.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to Novoda's Open Source projects
+
+We encourage everyone inside and outside Novoda to contribute to the projects using Github's pull requests.
+
+## Issuing a pull request
+
+Github makes it really easy to create a pull request (PR) against a repo. Just fork it, implement your changes and create a pull request back to the original repo.
+
+The PR should follow this format:
+
+  * The title of the PR should be a short sentence explaining the fix
+  * The PR description must contain at least two sections:
+    1. **The problem**: Explain what's the bug you're trying to solve or the missing feature you're trying to add with this PR.
+    2. **The solution**: Explain the fix or feature you've implemented in the PR. If this PR caused any UI change, then you should include screenshots or gifs showing how it looked before and after the change. See https://guides.github.com/features/mastering-markdown/ to create great looking markdown tables for showing your UI changes.
+    3. Feel free to include funny memes or gifs ;)
```

Comment:
Taken from `novoda/novoda`. My thoughts are that the base probably needs updating too 😞